explore-reports-and-create-documentation/farm-explorer/farm-dashboard.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Farm Dashboard
+description: This article describes Farm Dashboard and its features.
+Author: Janko Ban
+date: 14/12/2018.
+---
+Farm dashboard is giving you an overview of the farm configuration, drillable to specific detailed reports.
+
+The following farm configurations are visible:
```

Comment:
Napisati kod svih tileova na koje točno reporte drilla, i, ako je potrebno, koji filteri su preselektirani. Npr. kod Databases i site collections ne piše na koji report ide.